.circleci/config.yml.in

```diff
@@ -171,6 +258,10 @@ jobs:
           name: {{prefix}}binary_{{os}}_{{btype}}_py{{python_version}}{{ "u" if unicode }}_{{cu_version}}
           python_version: "{{python_version}}"
           cu_version: "{{cu_version}}"
+          filters:
+            branches:
+              only:
+                - master
```

Comment:
You probably don't want to actually do this, as it will prevent you from easily doing binary builds by opening PRs.

Reply:
if I want only a subset of the builds to run, how would you recommend me doing this, with an approach similar to PyTorch [`should_run_job.sh`](https://github.com/pytorch/pytorch/blob/e59bbc82a0906708adff30a1258b2441e806964c/.circleci/scripts/should_run_job.sh) ?

Reply:
In a vacuum, what I would have expected to see is that by default there is some limited set of builds that run on master, but then there is an easy way to turn on all of the builds by editing CircleCI config, if you need it. That's what the `{%- if True %-}` block does today.

You can do a `should_run_job.sh` strategy, but if you enable too many builds by default on a PR, or too many builds which have exactly equal finish times, you'll hit GitHub rate limiting. We had exactly this problem. Another way to solve this problem is to use the parametrized jobs endpoint to trigger jobs yourself; ask @suo about it, he's been poking at it on main pytorch/pytorch repo.